docs/docs/global-css.md

```diff
@@ -0,0 +1,127 @@
+
+To create a shared layout with global styles, start by creating a new Gatsby site with the [hello world starter](https://github.com/gatsbyjs/gatsby-starter-hello-world).
+
+```shell
+gatsby new global-styles https://github.com/gatsbyjs/gatsby-starter-hello-world
+```
+
+Open your new site in your code editor and create a new directory at `/src/components`. Inside, create two new files:
+
+```diff
+  global-styles/
+  └───src/
+      └───components/
++     │   │─  layout.js
++     │   └─  layout.css
+      │
+      └───pages/
+          └─  index.js
+```
+
+Inside `src/components/layout.css`, add some global styles:
+
+```css:title=src/components/layout.css
+div {
+  background: red;
+  color: white;
+}
+```
+
+In `src/components/layout.js`, include the stylesheet and export a layout component:
+
+```jsx:title=src/components/layout.js
+import React from "react"
+import "./layout.css"
+
+export default ({ children }) => <div>{children}</div>
+```
+
+Finally, update `src/pages/index.js` to use the new layout component:
+
+```jsx:title=src/pages/index.js
+import React from "react"
+import Layout from "../components/layout"
+
+export default () => <Layout>Hello world!</Layout>
+```
+
+Run `npm run develop` and you’ll see the global styles applied.
+
+![Global styles](./images/global-styles.png)
+
+## Adding global styles without layout component
+
+In some cases, using a shared layout component is not desirable. In these cases, you can include a global stylesheet using `gatsby-browser.js`.
+
+> **NOTE:** This approach does _not_ work with CSS-in-JS. Use shared components to share styles in CSS-in-JS.
+
+First, open a new terminal window and run the following commands to create a new default Gatsby site and start the development server:
+
+```shell
+gatsby new global-style-tutorial https://github.com/gatsbyjs/gatsby-starter-default
+cd global-style-tutorial
+npm run develop
+```
+
+Second, create a css file and define any styles you wish. An arbitrary example:
+
+```css:title=src/styles/global.css
+html {
+  background-color: lavenderblush;
+}
+
+a {
+  color: rebeccapurple;
+}
+```
+
+Then, include the stylesheet in your site's `gatsby-browser.js` file.
+
+> **NOTE:** This solution works when including css as those styles are extracted when building the JavaScript but not for css-in-js.
+> Including styles in a layout component or a global-styles.js is your best bet for that.
+
+```javascript:title=gatsby-browser.js
+import "./src/styles/global.css"
+
+// or:
+// require('./src/styles/global.css')
+```
+
+> _Note: You can use Node.js require or import syntax. Additionally, the placement of the example css file in a `src/styles` folder is arbitrary._
+
+You should see your global styles taking effect across your site:
+
+![Global styles example site](./images/global-styles-example.png)
+
+### Adding classes to components
+
+Since `class` is a reserved word in JavaScript, you'll have to use the `className` prop instead.
```

Comment:
```suggestion
Since `class` is a reserved word in JavaScript, you'll have to use the `className` prop instead, which will render as the browser-supported `class` attribute in your HTML output.
```